auditbeat/auditbeat.reference.yml

```diff
@@ -1173,6 +1173,10 @@ logging.files:
   # Unix epoch. Defaults to disabled.
   #interval: 0
 
+  # Rotate existing logs on startup rather than appending to the existing
+  # file. Defaults to true.
+  # rotateonopen: true
```

Comment:
It feels like this setting would be clearer using `start` or `startup` instead of `open`. I don't think it's obvious to the user what `open` means here.

I'm OK with using open internally (rotator already uses `open` for this concept).

Reply:
good point, changed open -> startup everywhere (I'd rather use the same name internally so it's easy to find)